### PR TITLE
Fixed the Notes Issue in docstring of IrsaClass

### PR DESCRIPTION
**Pull request by AyushYadav**
@eteq @bsipocz @keflavich  Have a look at the PR. Details at [#681 Issue](https://github.com/astropy/astroquery/issues/681)

### CODE REVIEW

File: astroquery/irsa/core.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def query_region(self, coordinates=None, catalog=None, spatial='Cone',
             details). Required if spatial is ``'Cone'`` or ``'Box'``. Optional
             if spatial is ``'Polygon'``.
         catalog : str
-            The catalog to be used (see the *Notes* section below).
+            The catalog to be used can be found using ``'Irsa.print_catalogs()'`` which returns a dictionary of catalog names with their description.
```
Comment (bsipocz) [inline astroquery/irsa/core.py]: Please wrap the lines around 80 chars, and you can also link to the API docs by using only single backticks as `` :meth:`~astroquery.irsa.IrsaClass.print_catalogs( )` `` (see examples below)

**Comment by bsipocz** [comment]
Thanks @AyushYadav. I had a minor inline comment, please address it and then I'll merge this.

**Comment by AyushYadav** [comment]
@bsipocz Is this [Fix](https://github.com/astropy/astroquery/pull/851/commits/c226311b95c408158515255f059c155c1c7328c4) ok for merge. I have added :meth:`~astroquery.irsa.IrsaClass.print_catalogs` instead of :meth:`~astroquery.irsa.IrsaClass.print_catalogs( )` by looking at the same use case from Line 177 in [core.py](https://github.com/astropy/astroquery/blob/master/astroquery/irsa/core.py). Let me know if this is ok.

**Comment by bsipocz** [comment]
Thanks @AyushYadav. 

In future please use a feature branch to open PRs from rather than your master branch. It will make your life easier when working on several issues parallel, or just want to keep up with the recent master somewhere. More tips and tricks can be found in the astropy developer guide: http://docs.astropy.org/en/latest/development/workflow/development_workflow.html#astropy-guidelines-for-git

**Pull request merged**